src/Compilers/CSharp/Portable/Symbols/Source/SourceParameterSymbolBase.cs

```diff
@@ -127,6 +127,11 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
             {
                 AddSynthesizedAttribute(ref attributes, moduleBuilder.SynthesizeNullableAttributeIfNecessary(this, GetNullableContextValue(), type));
             }
+
+            if (type.Type.ContainsErasedExtensionType())
+            {
+                AddSynthesizedAttribute(ref attributes, compilation.SynthesizeExtensionErasureAttribute(type.Type));
```

Comment:
>AddSynthesizedAttribute(ref attributes, compilation.SynthesizeExtensionErasureAttribute(type.Type)); [](http://example.com/codeflow?start=16&length=100)

If we are not dealing with adding this attribute definition, perhaps we should check its presence and report an error when it is unavailable?

Reply:
I left that as a PROTOTYPE issue in test for now. It doesn't seem worth it to add a check that will be removed later. I could synthesize the attribute definition now if you prefer.